exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -159,7 +171,11 @@ export async function handler (
       : undefined
     return printProjectCommands(manifest, rootManifest ?? undefined)
   }
-  if (scriptName !== 'start' && !manifest.scripts?.[scriptName]) {
+
+  const multiScriptSelectorRegExp = buildRegExpFromCommand(scriptName)
+  const specifiedScripts = multiScriptSelectorRegExp ? Object.keys(manifest.scripts ?? {}).filter(script => script.match(multiScriptSelectorRegExp)) : (!!manifest.scripts?.[scriptName] || scriptName === 'start') ? [scriptName] : []
```

Comment:
this line is really hard to understand. It would be better to move it to a function.

Reply:
in what order should the scripts be executed? Alphabetical maybe?

Reply:
Indeed. I moved this process into `getSpecifiedScripts` function f6fe3cf 👍

Reply:
> in what order should the scripts be executed? Alphabetical maybe?

In my understanding, `package.manifest.scripts` is not sorted by alphabetical order when run command handler runs, so I think execution order is simply the order of `scripts` filed in `package.json`. 

In my opinion, it is not necessary to sort the order of execution alphabetically. This is because some scripts depend on the order of execution, and if a user want to guarantee the order of execution order and if execution order is  sorted by alphabetical order, a user will need to express the order of the names of the keys in the script accordingly, which may be difficult to understand.

Ex)
In the case  a user want to execute script in the order of `build:js` -> `build:css`.

if execution order is sorted by alphabetical order:
a user need to write scripts filed like

```json
  “build:1-js”: “~~”,
  “build:2-css”: “~~”
```

if execution order is not sorted by alphabetical order:
a user need to write scripts filed just like 

```json
  “build:js”: “~~”,
  “build:css”: “~~”
```